lib/ansible/plugins/action/reboot.py

```diff
@@ -313,6 +305,14 @@ def run(self, tmp=None, task_vars=None):
             result['elapsed'] = elapsed.seconds
             return result
 
+        post_reboot_delay = int(self._task.args.get('post_reboot_delay', self._task.args.get('post_reboot_delay_sec', self.DEFAULT_POST_REBOOT_DELAY)))
+        if post_reboot_delay < 0:
+            post_reboot_delay = 0
+
+        if post_reboot_delay != 0:
+            display.vvv("%s: waiting an additional %d seconds" % (self._task.action, post_reboot_delay))
```

Comment:
Same thing here: need `post_reboot_delay_sec` here since `win_reboot` accepts that.

Reply:
This is being set a few lines higher with one of the values, so I think this one is correct.

Reply:
Sorry, I've been staring at this too long. :)

Reply:
No worries, I made my share of oversights in this case too :-)